draft-ietf-httpbis-http2-secondary-certs.md

```diff
@@ -563,20 +557,22 @@ Request-ID:
   certificate-related frames with this request.  The identifier MUST be unique
   in the session for the sender.
 
-Extension-Count and Extensions:
-: A list of certificate selection criteria, represented in a series of
-  `Extension` structures (see [I-D.ietf-tls-tls13] section 4.2). This criteria
-  MUST be used in certificate selection as described in {{I-D.ietf-tls-tls13}}.
-  The number of `Extension` structures is given by the 16-bit `Extension-Count`
-  field, which MAY be zero.
+Request:
+: An exported authenticator request, generated using the `request` API described
+  in [I-D.ietf-tls-exported-authenticator]. See {{exp-auth}} for more details on
+  the input to this API.
+
+### Exported Authenticator Request Characteristics {#exp-req}
 
-Some extensions used for certificate selection allow multiple values (e.g.
-oid_filters on Extended Key Usage). If the sender has included a non-empty
-Extensions list, the certificate MUST match all criteria specified by extensions
-the recipient recognizes. However, the recipient MUST ignore and skip any
-unrecognized certificate selection extensions.
+The Exported Authenticator `request` API defined in
+[I-D.ietf-tls-exported-authenticator] takes as input a set of desired
+certificate characteristics and a `certificate_request_context`. When generating
+exported authenticators for use with this extension, the
+`certificate_request_context` MUST be the two-octet Request-ID.
```

Comment:
Exported authenticators require that the certificate_request_context be unpredictable.  I think that we can address this by prefixing a random sequence with the two-octet Request-ID.  Then we can ensure that we have a mapping key, but also enough entropy to avoid guessing.

Reply:
Should we specify the length?

Reply:
We probably need a minimum size to stop someone from inducing lots of requests until the right number comes up (for the stolen credentials they have).